src/engine/mailprivacy_gnupg.c

```diff
@@ -2837,17 +2823,17 @@ static int mailprivacy_gnupg_init_lock_done = 0;
 #endif
 static chash * encryption_id_hash = NULL;
 
-static void mailprivacy_gnupg_init_lock(void)
```

Comment:
GCC warned that mailprivacy_gnupg_init_lock() wasn't used anywhere, so I would have removed it; but then I saw that this code is executed `#elif (defined WIN32)`, so I thought "I can't test it under Windows, let's move the whole function under this `#elif`": one less warning under Linux, no change under Windows.